File: accepted-solutions/aaah/aaah.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    string jon;
    string doctor;

    getline(cin, jon);
    getline(cin, doctor);

    cout << (jon.size() >= doctor.size() ? "go" : "no") << endl;

    return 0;
}

File: accepted-solutions/aaah/aaah.java
import java.util.Scanner;

public class aaah {
    public static void main(String[] args) {
        Scanner cin = new Scanner(System.in);

        String jon = cin.nextLine();
        String doctor = cin.nextLine();

        System.out.println(jon.length() >= doctor.length() ? "go" : "no");
    }
}
